src/utils/slack.js

```diff
@@ -1,41 +0,0 @@
-const token = process.env.SLACK_TOKEN;
-
```

Comment:
Just wondering, what was the reason these were moved from a separate file to within the component?

I think this approach would have been good, the logic could be within redux actions and reducers and your components could be dumb simply passing in the desired props as parameters.

Would probably make it easier to test as well!